mc/Mongo-Core.package/MongoQueryBatch.class/instance/initializeWithCollectionName.documents.cursorId..st

```diff
@@ -0,0 +1,8 @@
+initialization
+initializeWithCollectionName: aString documents: aCollection cursorId: anInteger
+	self initialize.
+
+	self flag: #todo. "This code removes the db name, and it's a bit strange to have it here."	
+	collectionName := aString readStream upTo: $.; upToEnd.
```

Comment:
What are the alternatives?

Reply:
Didn't find another occurrence in the package but may be more general... it splits the db name prefix from the full scoped collection name. At least, an alternative is to store the full name in the object and implement an accessor like `nonScopedCollectionName` that trims, instead of having it in the initiailze.